cstrike/addons/amxmodx/configs/plugins/ChatAdditions/ChatAdditions_core.cfg

```diff
@@ -31,4 +31,17 @@ ca_log_level "1"
 // Maximum: "1.000000"
 ca_update_notify "1"
 
+// The time in days after which the log files should be deleted.
+// -
+// Default: "7"
+// Minimum: "0.000000"
+ca_log_autodelete_time "7"
+
+// Auto delete log files older than X days?
+// -
+// Default: "1"
+// Minimum: "0.000000"
+// Maximum: "1.000000"
+ca_log_autodelete "1"
```

Comment:
There is no need to create another cvar. Just make it so that if `ca_log_autodelete_time 0`, logs are not deleted